plugins/default/lib/disk_usage.py

```diff
@@ -1,35 +1,97 @@
 #!/usr/bin/env python
 import re
 import sys
+import json
+import argparse
 from subprocess import CalledProcessError
+from collections import Counter
 
-from openshift import oc
+import openshift
 import nagios
 
 
-check_disk_cmd = ("df", "--output=target,pcent,ipcent")
+check_disk_cmd = ("df", "--output=pcent,ipcent,target")
 # Example output:
 # /etc/hosts      19%    8%
-check_disk_output_pattern = re.compile(r"([\w/]+)\s+(\d+)%\s+(\d+)%")
+check_disk_output_pattern = re.compile(
+    r"\s*(\d+)%\s+(\d+)%\s+(.+)$")
```

Comment:
Yeah, I never used `autopep8` myself, PEP8 is loose in several points, where "common sense" is suggested.
I suggested using the tool to give us some coding standard, and save some time in code review.